omen_light.cc

```diff
@@ -122,14 +123,27 @@ bool SendReport(const Report& report) {
   return true;
 }
 
-bool ParseLed(const std::string& arg, Report::Led* led) {
+bool ParseLed(const std::string& arg, uint8_t* led) {
   if (arg == "front") {
-    *led = Report::Led::FRONT;
+    *led = 1;
   } else if (arg == "chase") {
-    *led = Report::Led::CHASE;
+    *led = 2;
   } else {
-    std::cerr << "unknown led: " << arg << std::endl;
-    return false;
+    try
+    {
```

Comment:
I use google style for clang format. Maybe follow the same style here?

Reply:
Good call